Doc/whatsnew/3.7.rst

```diff
@@ -483,138 +782,458 @@ The :func:`~binascii.b2a_uu` function now accepts an optional *backtick*
 keyword argument.  When it's true, zeros are represented by ``'`'``
 instead of spaces.  (Contributed by Xiang Zhang in :issue:`30103`.)
 
+
 calendar
 --------
 
 The class :class:`~calendar.HTMLCalendar` has new class attributes which ease
 the customisation of the CSS classes in the produced HTML calendar.
 (Contributed by Oz Tiram in :issue:`30095`.)
 
+The :func:`~calendar.weekday` function now accepts any value for its *year*
+argument (previously only values from ``1`` to ``9999`` were accepted).
+(Contributed by Mark Gollahon in :issue:`28281`.)
+
+:meth:`Calendar.itermonthdates <calendar.Calendar.itermonthdates>`
```

Comment:
Mention new methods first. The change in `itermonthdates()` is rather a bug fix.